packages/kotti-ui/source/kotti-field-file-upload/components/TakePhoto/Error.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+	<KtModal isOpen size="lg">
+		<div slot="header">
+			<h2 v-text="translations.label.error" />
+		</div>
+		<div slot="body">
+			<div v-if="error" v-text="errorText" />
+		</div>
+		<div slot="footer">
+			<KtButton
+				class="mr-4"
+				:data-test="dataTest ? `${dataTest}.retakePhoto` : undefined"
+				:label="translations.button.retry"
```

Comment:
i'm not really sure this would be reactive

can you try changing translations and seeing if the values recompute on demand?

Reply:
Tested, works fine on demand with all translations